### DISCUSSION TITLE
Assignment Update

### DISCUSSION THREAD

**Discussion by thawkes70** [thawkes70/Tafe22S219#3] [Announcements]
Hi Chitra,

I have created the development branch from the main branch.  I have created the main menu in VS2019 and committed the files back to development,  GitHub has been synced to my repository so we should be all good to go with Section C of the assignment.

Cheers

Tony

**Comment by thawkes70** [comment]
Oh I cant remember who was doing which feature?  I'm not fussed either way.

**Reply by Chitusnair** [reply]
Hi Tony,

Thanks so far. I will be working on the Mortgage Calculator. So you can work on the Currency Converter.

**Reply by thawkes70** [reply]
Ok cool, now how do you C# and Xaml again?  Ha its been a while for me, better dig out some old assignments.

**Reply by Chitusnair** [reply]
Same here. I will let you know as soon as I figure out.

**Reply by Chitusnair** [reply]
Seems like you figured out the Xaml and C#:)
